includes/widgets/google-maps.php

```diff
@@ -150,7 +150,16 @@ protected function register_controls() {
 						'min' => 40,
 						'max' => 1440,
 					],
+					'vh' => [
+						'min' => 0,
+						'max' => 100,
+					],
+					'vw' => [
+						'min' => 0,
+						'max' => 100,
+					],
```

Comment:
`vw` unit is usually irrelevance for height values. Is there a specific reason to add it here?

Reply:
@avivu?

Reply:
We can remove the VW